wp-content/themes/pub/wporg-learn-2024/src/style/style.scss

```diff
@@ -39,3 +39,28 @@ body {
 		padding-top: unset !important;
 	}
 }
+
+.wporg-learn-sidebar-meta-info {
+	.wp-block-sensei-lms-button-take-course {
+		margin-top: 0;
+		margin-bottom: var(--wp--preset--spacing--20);
+
+		> button {
+			line-height: 1;
+		}
+	}
```

Comment:
I would put anything sensei specific that we can't do with block markup in _sensei.scss, or a  `wp-block-sensei-lms-button-take-course.scss` file

Reply:
b2b192e

Have tried moving the margin top and bottom to the markup as well, but the styles added [here](https://github.com/WordPress/Learn/blob/fe056e2de03365a076c31976b5651fa3fce0b045/wp-content/themes/pub/wporg-learn-2024/patterns/sidebar-meta-info.php#L23) only apply to the highlighted div rather than the div being pointed to, which doesn't really overwrite the default margin.

![image](https://github.com/WordPress/Learn/assets/18050944/945c0d7b-cf6a-4ff0-84fc-e0cb8a227e80)

Reply:
b2b192e